examples/mongodbatlas_third_party_integration/datadog/variables.tf

```
@@ -24,3 +24,15 @@ variable "cluster_name" {
   default     = "datadog-test-cluster"
   type        = string
 }
+
+variable "send_collection_latency_metrics" {
+  description = "Send collection latency metrics"
```

```suggestion
  description = "Send collection latency metrics (only for Datadog integrations)"
```

nit: add comment, also for send_database_metrics var

---

added